### ISSUE TITLE
Please provide some screenshots or gif

### ISSUE THREAD

**Issue by chakrihacker** [eaceto/react-native-youtube-player#2]

**Comment by eaceto** [comment]
Hello... sorry for writing so late. I'll do so if you are still insterested.

**Comment by chakrihacker** [comment]
yes please

**Comment by eaceto** [comment]
I'll keep you posted. Thanks!

**Comment by eaceto** [comment]
Here you have two examples when using in landscape mode. Also I have uploaded a youtube video using it on portrait. 

https://www.youtube.com/watch?v=rGq_y5pF0xA&feature=youtu.be

The readme was updated (some changes in iOS setup). Let me know if you have issues integrating this lib, and I will gladly help you.



<img width="848" alt="screen shot 2018-10-24 at 20 11 05" src="https://user-images.githubusercontent.com/870528/47466806-78ce8980-d7c9-11e8-9531-5ae9d0a20a10.png">

![simulator screen shot - iphone 6 plus - 2018-10-24 at 20 11 10](https://user-images.githubusercontent.com/870528/47466807-79672000-d7c9-11e8-8c46-89957b531707.png)
